lib/phoenix_live_reload/live_reloader.ex

```diff
@@ -89,9 +89,13 @@ defmodule Phoenix.LiveReloader do
     end
   end
 
-  defp before_send_inject_reloader(conn, endpoint) do
+  defp before_send_inject_reloader(%{resp_body: resp_body} = conn, endpoint) do
     register_before_send conn, fn conn ->
-      resp_body = to_string(conn.resp_body)
+      resp_body = if is_list(resp_body) do
```

Comment:
We don't need the `if`. We can just always use IO.iodata_to_string/1. Thank you!